### PR TITLE
Unify themes reviewers queue into a single queue

### PR DESCRIPTION
**Pull request by diox**
Fixes: https://github.com/mozilla/addons/issues/14935

### Testing

- Submit a new theme (using the theme creation wizard or not), find it in reviewer tools new "Themes" queue, approve it, submit a new version of the theme after that
- Submit another new theme
- You should see both themes in the reviewer tools themes queue

### CODE REVIEW

File: src/olympia/reviewers/views.py [hunk 1]
```diff
@@ -233,12 +232,8 @@ def dashboard(request):
     ):
         sections['Themes'] = [
             (
-                'New ({0})'.format(queue_counts['theme_nominated']),
-                reverse('reviewers.queue_theme_nominated'),
-            ),
-            (
-                'Updates ({0})'.format(queue_counts['theme_pending']),
-                reverse('reviewers.queue_theme_pending'),
+                'Manual Review ({0})'.format(queue_counts['theme']),
```
Comment (eviljeff) [inline src/olympia/reviewers/views.py:235]: Do we want to call it "Manual Review" when all themes are (right now) pre-review, so all manually reviewed? I don't have a great alternative title though... just "Review"?  "Awaiting Review"? 🤔

**Review by eviljeff — APPROVED** [review]
Wow.... I thought it would need some significant refactoring/reworking.  Yay past us for writing generic enough code I guess?

**Comment by eviljeff** [comment]
I think we should confirm with Scott before merging this so he has a chance to notify the theme reviewers of the upcoming change.

File: src/olympia/reviewers/views.py (hunk 1 above)
Comment (diox) [reply src/olympia/reviewers/views.py:235]: `Awaiting Review` is probably better than `Manual Review` yeah.

**Comment by diox** [comment]
We have decided to wait until after MozWeek.

**Pull request merged**